luaui/Widgets/cmd_unitgroup_remove.lua

```diff
@@ -0,0 +1,28 @@
+function widget:GetInfo()
+  return {
+    name      = "Unit Groups - Remove units from a group",
+    desc      = "Remove units from a unit group on action 'groupclear'",
```

Comment:
Isn't this already covered by CTRL+`? I use it all the time to remove selected units from their current group.

Reply:
Yeah you're binding the wrong action. It's not `groupclear` it's `group unset`, and it's already bound by default
https://github.com/beyond-all-reason/Beyond-All-Reason/blob/295cf560c8f7c8b751ee0fd13ec19549061441b6/luaui/configs/hotkeys/default_keys.txt#L311
https://github.com/beyond-all-reason/Beyond-All-Reason/blob/295cf560c8f7c8b751ee0fd13ec19549061441b6/luaui/configs/hotkeys/grid_keys.txt#L154

Reply:
Okay thanks, that works. For reference:
https://springrts.com/phpbb/viewtopic.php?f=1&t=44462

Will remove the widget 👍